src/tests/FunctionalTests/iOS/Device/AOT-LLVM/iOS.Device.Aot-Llvm.Test.csproj

```diff
@@ -1,17 +1,29 @@
 <Project Sdk="Microsoft.NET.Sdk" TreatAsLocalProperty="MonoForceInterpreter">
-
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <MonoForceInterpreter>false</MonoForceInterpreter>
-    <RunAOTCompilation>true</RunAOTCompilation>
+    <OutputPath>bin</OutputPath>
     <TestRuntime>true</TestRuntime>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <RuntimeIdentifier>$(TargetOS)-$(TargetArchitecture)</RuntimeIdentifier>
     <TargetOS Condition="'$(TargetOS)' == ''">ios</TargetOS>
-    <MainLibraryFileName>iOS.Device.Aot-Llvm.Test.dll</MainLibraryFileName>
     <IncludesTestRunner>false</IncludesTestRunner>
     <ExpectedExitCode>42</ExpectedExitCode>
-    <EnableAggressiveTrimming Condition="'$(EnableAggressiveTrimming)' == ''">true</EnableAggressiveTrimming>
+    <SelfContained>true</SelfContained>
+    <UseConsoleUITemplate>true</UseConsoleUITemplate>
+    <ShouldILStrip Condition="'$(ArchiveTests)' != 'true' and '$(RunAOTCompilation)' == 'true' and '$(MonoForceInterpreter)' != 'true'">true</ShouldILStrip>
```

Comment:
Do we need `'$(MonoForceInterpreter)' != 'true'` condition? 
For this test `MonoForceInterpreter` was always `false`.

Reply:
The functional tests are executed on simulators as well using JustInterp AOT mode.

Reply:
My error, there is a different directory for simulators. This should be merged since they don't differ.

Reply:
Additionally, not sure about `EnableAggressiveTrimming`, should it also be set for Mono builds?

Reply:
There was an issue with EnableAggressiveTrimming on the Helix. I will revert the change to test the CI.

Reply:
A lot of methods can't be found during AOT compilation when aggressive trimming is enabled.

https://[helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-87773-merge-473f623714b94e4bb7/iOS.Device.Aot.Test/1/console.e2679643.log?helixlogtype=result](https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-87773-merge-473f623714b94e4bb7/iOS.Device.Aot.Test/1/console.e2679643.log?helixlogtype=result)

Reply:
I've created a tracking issue: https://github.com/dotnet/runtime/issues/90025